File: Syncfusion_Sample/Windows/Grid.Windows/Samples/2.0/Product Showcase/Excel Like UI Demo/CS/Unhide.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using Syncfusion.Windows.Forms;

namespace ExcelLikeUI_2005
{
	/// <summary>
	/// Summary description for Unhide.
	/// </summary>
	public class Unhide : Office2007Form
	{
		private ButtonAdv okButton;
		private ButtonAdv cancelButton;
		internal System.Windows.Forms.ListBox sheetListBox;
		/// <summary>
		/// Required designer variable.
		/// </summary>
		private System.ComponentModel.Container components = null;

		public Unhide()
		{
			//
			// Required for Windows Form Designer support
			//
			InitializeComponent();

            this.AcceptButton.DialogResult = DialogResult.OK;

			//
			// TODO: Add any constructor code after InitializeComponent call
			//
			
		}

		/// <summary>
		/// Clean up any resources being used.
		/// </summary>
		protected override void Dispose( bool disposing )
		{
			if( disposing )
			{
				if(components != null)
				{
					components.Dispose();
				}
			}
			base.Dispose( disposing );
		}

		#region Windows Form Designer generated code
		/// <summary>
		/// Required method for Designer support - do not modify
		/// the contents of this method with the code editor.
		/// </summary>
		private void InitializeComponent()
		{
            this.sheetListBox = new System.Windows.Forms.ListBox();
            this.okButton = new Syncfusion.Windows.Forms.ButtonAdv();
            this.cancelButton = new Syncfusion.Windows.Forms.ButtonAdv();
            this.SuspendLayout();
            // 
            // sheetListBox
            // 
            this.sheetListBox.Anchor = ((System.Windows.Forms.AnchorStyles)((((System.Windows.Forms.AnchorStyles.Top | System.Windows.Forms.AnchorStyles.Bottom)
                        | System.Windows.Forms.AnchorStyles.Left)
                        | System.Windows.Forms.AnchorStyles.Right)));
            this.sheetListBox.Location = new System.Drawing.Point(8, 16);
            this.sheetListBox.Name = "sheetListBox";
            this.sheetListBox.Size = new System.Drawing.Size(216, 108);
            this.sheetListBox.TabIndex = 0;
            // 
            // okButton
            // 
            this.okButton.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Right)));
            this.okButton.Appearance = Syncfusion.Windows.Forms.ButtonAppearance.Office2007;
            this.okButton.DialogResult = System.Windows.Forms.DialogResult.OK;
            this.okButton.Location = new System.Drawing.Point(45, 147);
            this.okButton.Name = "okButton";
            this.okButton.Size = new System.Drawing.Size(75, 23);
            this.okButton.TabIndex = 1;
            this.okButton.Text = "OK";
            this.okButton.UseVisualStyle = true;
            // 
            // cancelButton
            // 
            this.cancelButton.Anchor = ((System.Windows.Forms.AnchorStyles)((System.Windows.Forms.AnchorStyles.Bottom | System.Windows.Forms.AnchorStyles.Right)));
            this.cancelButton.Appearance = Syncfusion.Windows.Forms.ButtonAppearance.Office2007;
            this.cancelButton.DialogResult = System.Windows.Forms.DialogResult.Cancel;
            this.cancelButton.Location = new System.Drawing.Point(149, 147);
            this.cancelButton.Name = "cancelButton";
            this.cancelButton.Size = new System.Drawing.Size(75, 23);
            this.cancelButton.TabIndex = 2;
            this.cancelButton.Text = "Cancel";
            this.cancelButton.UseVisualStyle = true;
            // 
            // Unhide
            // 
            this.AcceptButton = this.okButton;
            this.AutoScaleBaseSize = new System.Drawing.Size(5, 13);
            this.BackColor = System.Drawing.Color.FromArgb(((int)(((byte)(246)))), ((int)(((byte)(249)))), ((int)(((byte)(254)))));
            this.CancelButton = this.cancelButton;
            this.ClientSize = new System.Drawing.Size(236, 182);
            this.Controls.Add(this.cancelButton);
            this.Controls.Add(this.okButton);
            this.Controls.Add(this.sheetListBox);
            this.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog;
            this.MaximizeBox = false;
            this.MaximumSize = new System.Drawing.Size(248, 248);
            this.MinimizeBox = false;
            this.Name = "Unhide";
            this.ShowIcon = false;
            this.StartPosition = System.Windows.Forms.FormStartPosition.CenterParent;
            this.Text = "Unhide";
            this.ResumeLayout(false);

		}
		#endregion
	}
}

File: Syncfusion_Sample/Windows/Grid.Windows/Samples/2.0/Product Showcase/Excel Like UI Demo/VB/Unhide.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Drawing
Imports System.Collections
Imports System.ComponentModel
Imports System.Windows.Forms
Imports Syncfusion.Windows.Forms

Namespace ExcelLikeUI_2005
	''' <summary>
	''' Summary description for Unhide.
	''' </summary>
	Public Class Unhide
		Inherits Office2007Form
		Private okButton As ButtonAdv
        Private Shadows cancelButton As ButtonAdv
		Friend sheetListBox As System.Windows.Forms.ListBox
		''' <summary>
		''' Required designer variable.
		''' </summary>
		Private components As System.ComponentModel.Container = Nothing

		Public Sub New()
			'
			' Required for Windows Form Designer support
			'
			InitializeComponent()

			Me.AcceptButton.DialogResult = System.Windows.Forms.DialogResult.OK

			'
			' TODO: Add any constructor code after InitializeComponent call
			'

		End Sub

		''' <summary>
		''' Clean up any resources being used.
		''' </summary>
		Protected Overrides Overloads Sub Dispose(ByVal disposing As Boolean)
			If disposing Then
				If Not components Is Nothing Then
					components.Dispose()
				End If
			End If
			MyBase.Dispose(disposing)
		End Sub

		#Region "Windows Form Designer generated code"
		''' <summary>
		''' Required method for Designer support - do not modify
		''' the contents of this method with the code editor.
		''' </summary>
		Private Sub InitializeComponent()
			Me.sheetListBox = New System.Windows.Forms.ListBox()
			Me.okButton = New Syncfusion.Windows.Forms.ButtonAdv()
			Me.cancelButton = New Syncfusion.Windows.Forms.ButtonAdv()
			Me.SuspendLayout()
			' 
			' sheetListBox
			' 
			Me.sheetListBox.Anchor = (CType((((System.Windows.Forms.AnchorStyles.Top Or System.Windows.Forms.AnchorStyles.Bottom) Or System.Windows.Forms.AnchorStyles.Left) Or System.Windows.Forms.AnchorStyles.Right), System.Windows.Forms.AnchorStyles))
			Me.sheetListBox.Location = New System.Drawing.Point(8, 16)
			Me.sheetListBox.Name = "sheetListBox"
			Me.sheetListBox.Size = New System.Drawing.Size(216, 108)
			Me.sheetListBox.TabIndex = 0
			' 
			' okButton
			' 
			Me.okButton.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Right), System.Windows.Forms.AnchorStyles))
			Me.okButton.Appearance = Syncfusion.Windows.Forms.ButtonAppearance.Office2007
			Me.okButton.DialogResult = System.Windows.Forms.DialogResult.OK
			Me.okButton.Location = New System.Drawing.Point(45, 147)
			Me.okButton.Name = "okButton"
			Me.okButton.Size = New System.Drawing.Size(75, 23)
			Me.okButton.TabIndex = 1
			Me.okButton.Text = "OK"
			Me.okButton.UseVisualStyle = True
			' 
			' cancelButton
			' 
			Me.cancelButton.Anchor = (CType((System.Windows.Forms.AnchorStyles.Bottom Or System.Windows.Forms.AnchorStyles.Right), System.Windows.Forms.AnchorStyles))
			Me.cancelButton.Appearance = Syncfusion.Windows.Forms.ButtonAppearance.Office2007
			Me.cancelButton.DialogResult = System.Windows.Forms.DialogResult.Cancel
			Me.cancelButton.Location = New System.Drawing.Point(149, 147)
			Me.cancelButton.Name = "cancelButton"
			Me.cancelButton.Size = New System.Drawing.Size(75, 23)
			Me.cancelButton.TabIndex = 2
			Me.cancelButton.Text = "Cancel"
			Me.cancelButton.UseVisualStyle = True
			' 
			' Unhide
			' 
			Me.AcceptButton = Me.okButton
			Me.AutoScaleBaseSize = New System.Drawing.Size(5, 13)
			Me.BackColor = System.Drawing.Color.FromArgb((CInt(Fix((CByte(246))))), (CInt(Fix((CByte(249))))), (CInt(Fix((CByte(254))))))
			Me.CancelButton = Me.cancelButton
			Me.ClientSize = New System.Drawing.Size(236, 182)
			Me.Controls.Add(Me.cancelButton)
			Me.Controls.Add(Me.okButton)
			Me.Controls.Add(Me.sheetListBox)
			Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.FixedDialog
			Me.MaximizeBox = False
			Me.MaximumSize = New System.Drawing.Size(248, 248)
			Me.MinimizeBox = False
			Me.Name = "Unhide"
			Me.ShowIcon = False
			Me.StartPosition = System.Windows.Forms.FormStartPosition.CenterParent
			Me.Text = "Unhide"
			Me.ResumeLayout(False)

		End Sub
		#End Region
	End Class
End Namespace
